src/main/java/org/fundacionjala/coding/Ruber/Kata1.java

```diff
@@ -0,0 +1,55 @@
+package org.fundacionjala.coding.Ruber;
+
+/**
+ * Created by Ruber Cuellar on 2/24/2017.
+ */
+public final class Kata1 {
+    private static final int NUMBERLIMIT = 5;
+    /**
+     * Constructor.
+     */
+    private Kata1() {
+    }
+
+    /**
+     * This method divides a word.
+     * @param text The test to divice
+     * @return String[] with the words
+     */
+    public static String[] diviceWords(final String text) {
+        return text.split(" ");
+    }
+
+    /**
+     * This method counts the number of characters.
+     * @param word The word to count
+     * @return boolean with the verification
+     */
+    public static boolean count(final String word) {
```

Comment:
I think this method is not necessary
you can use it directly in the main method